gallery/workbook/Azure Monitor.json

```diff
@@ -74,10 +74,15 @@
 			"name": "Synapse",
 			"templates": [
 				{
-					"id": "Workbooks/Synapse/Streaming",
+					"id": "Workbooks/Synapse/Infrastructure",
```

Comment:
lets be more specific in calling in Synapse/Spark/Infrastructure.. and name is Spark Infrastructure Metrics

Reply:
Thanks, I renamed them. Also removed the 'Metrics" suffix, seems redundant and other workbooks don't have that suffix as well